pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -251,8 +254,14 @@ func (o *capability) Execute(ctx context.Context, r capabilities.CapabilityReque
 		}
 
 		response := <-ch
+		outputLenBytes := values.ByteSizeOfMap(response.Value)
```

Comment:
@patrickhuie19 Cohesion is the name of the game :) Can you reduce cognitive overhead by removing the interface and just inlining what you need here?

Your interface + then calling it could just become

```
Metadata: capabilities.ResponseMetadata{
				Metering: []capabilities.MeteringNodeDetail{
					{SpendUnit: metering.NewPayloadMeteringUnit(), SpendValue: fmt.Sprintf("%d", inputLenBytes+outputLenBytes)},				
```